tools/fileinfo/features/pe-loader-corruptions/test.py

```diff
@@ -32,6 +32,7 @@ def test_corrupted_pe(self):
         assert self.fileinfo.succeeded
         self.assertEqual(self.fileinfo.output["loaderError"]["code"], 2)
         self.assertEqual(self.fileinfo.output["loaderError"]["code_text"], 'LDR_ERROR_E_LFANEW_UNALIGNED')
+        self.assertEqual(self.fileinfo.output["loaderError"]["loadable_anyway"], '0')
```

Comment:
Instead of storing the value as a number in string, shouldn't we just store it as `true`/`false`? JSON supports the boolean data type, so we should consider using it. @metthal What do you think?

Reply:
I'll replace it with "true" / "false" (oncluding quotation marks) to make it consistent with existing JSON variables